queries/pokemon_query.go

```diff
@@ -0,0 +1,39 @@
+package queries
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/rromodev/academy-go-q32021/models"
+)
+
+func GetPokemonById(id int) models.Pokemon {
+	pokemon := models.Pokemon{}
+
+	file, err := os.Open("./data.csv")
+	if err != nil {
+		fmt.Println(err)
```

Comment:
To handle errors, it's ok if you log what happened, but if the rest of the flow depends on the opened file and there was a problem opening it, then the rest of this function will break. So, you would like to return here, and return an error too.